ChangeLog.d/00README.md

```diff
@@ -3,6 +3,28 @@
 This directory contains changelog entries that have not yet been merged
 to the changelog file ([`../ChangeLog`](../ChangeLog)).
 
+## What requires a changelog entry?
+
+Write a changelog entry if there is a user-visible change. This includes:
+
+* Bug fixes: fixing a security hole, fixing broken behavior, fixing
+  the build in some configuration or on some platform, ...
+* New features in the library or new platform support.
+* Changes in existing behavior. These should be rare. Changes in features
+  that are documented as experimental may or may not be announced, depending
+  on the extent of the change and how widely we expect the feature to be used.
+
+We generally don't include changelog entries for:
+
+* Documentation improvements.
+* Performance improvements, unless they are particularly significant.
+* Changes that don't impact library users directly, for example new tests or
```

Comment:
"Changes that don't impact library users directly": already said in line 8 thus just:
"* New program or unit tests or changes to the test framework" here ?